.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 
-**Does the wallet support layer 2 networks?**
+**Spam protection?**
 
-<!-- Please list which L2 solutions the wallet supports. -->
+<!-- Does the wallet employ any practices to warn users against potential spam (e.g. when interacting with suspicious accounts/contracts)? -->
 
-**Can the wallet be used with arbitrary Ethereum RPC endpoint?**
+### Features
 
-<!-- Does wallet allow user to use the wallet with own Eth client? -->
+**Does the wallet have hardware wallet support?**
 
-**Does the wallet have fiat on-ramps?**
+<!-- Please provide information on how a user can connect a hardware wallet to this wallet. -->
 
-<!-- Does the wallet subsidise transaction fees for all or some wallet activity? -->
+**Does the wallet support WalletConnect?**
 
-**Does the wallet allow users to explore dapps?**
+<!-- Please provide information on if the wallet supports WalletConnect. -->
 
-<!-- Is the wallet integrated with WalletConnect? Does it have a dapp browser? Or can users connect to a dapp using the wallet?  -->
+**Does the wallet support importing Ethereum RPC endpoints?**
 
-**Does the wallet have integrated defi/financial tools?**
+<!-- Please provide documentation on how a user can import an Ethereum RPC into the wallet. -->
 
-<!-- Can users borrow/earn/lend assets directly from a screen in the wallet?  -->
+**Does the wallet support viewing and interacting with NFTs?**
 
-**Can a user withdraw to their card?**
+<!-- Please provide information on the experience of NFTs in the wallet. -->
 
-<!-- Can a user cash out their crypto assets directly to a card or bank account? -->
+**Does the wallet support connecting to Ethereum applications?**
 
-**Does the wallet offer limits protection?**
+<!-- Please provide documentation for how users connect to applications. List examples (ie. connect wallet to dapp, in wallet browser, etc.) -->
 
-<!-- Can a user set a limit for transfers to protect their assets? -->
+**Does the wallet support staking directly?**
 
-**Does the wallet allow high-volume purchases?**
+<!-- Please provide documentation on direct staking this wallet supports. -->
 
-<!-- Can a user buy more than $2000 worth of crypto in one go? -->
+**Does the wallet support swaps directly?**
 
-**Does the wallet have an integrated token swap?**
+<!-- Please provide documentation on swaps. -->
 
-<!-- Can a user swap ETH for other tokens from within a screen in the wallet? -->
+**Does the wallet support multi-chain networks?**
 
-**Is the wallet a multi-signature wallet?**
+<!-- Please provide documentation on multi-chain networks this wallet supports. -->
 
-<!-- Can a user opt to require multiple signatures to authorise transactions? -->
+**Does the wallet allow the user to customize gas fees?**
 
-**Wallet title**
+<!-- Please provide documentation on how users can customize gas fees for transactions. -->
 
-<!-- Please provide the official name of the wallet -->
+**Does the wallet support ENS?**
```

Comment:
I'm not exactly sure what this means - the wallet supports sending transactions to an ENS address? Perhaps we can make this more clear.

Reply:
@samajammin Assuming it means that you can send funds by typing in someones .eth address
```suggestion
**Does the wallet support sending to an ENS address?**
```
@minimalsm No, receiving funds is a passive process

Reply:
Or maybe 
```suggestion
**Does the wallet support sending to .eth addresses?**
```